### ISSUE TITLE
Post-Workshop Evaluation - Task Force

### ISSUE THREAD

**Issue by arockenberger** [uio-carpentry/organisational#33]
Hei!

I want to ask for volunteers who would like to help me with revising the currently used post-workshop evaluation form.

It would be nice if 2-3 of you would like to join a Task Force and take a look at the survey, the questions, and where we get weird responses.

If you also have experience with doing this kind of research (surveys, evaluations), your input will be much appreciated!

Respond to this Issue if you want to be part of the Task Force!

**Comment by naoe-tatara** [comment]
Now the Carpentry-central organized surveys (pre- and post-) are available, so if there are anything that UiO should know on top of them.

**Comment by lexnederbragt** [comment]
I support this but would hope someone else then me could be on the Task Force. It could be a task for the next board, alternatively, we ask some community members to help out.

**Comment by arockenberger** [comment]
Noted!
I'd rather it be a community effort than an actual task for the Board – for the time being I will continue sending out the old feedback form, too.
I can un-assign you if you want to!

**Comment by arockenberger** [comment]
In the Board Meeting on Dec 9, 2020 (https://github.com/uio-carpentry/organisational/blob/master/meetings/201209_board_meeting.md), we discussed what to do with workshop evaluations. (1) It was decided to use the official post-workshop survey for all workshops that are registered with The Carpentries and feedback the results to our community (via the regional coordinator or via direct contact with The Carpentries central). Instructors are asked to allot 5-10min at the end of the workshop and request learners to take the survey. (2) The Library will shorten the post-workshop evaluation questionnaire and focus on feedback regarding hosting, other topics for workshops etc.
(3) Direct feedback for instructors is collected via Sticky-Note-Feedback. After the instructors have gone through the notes, the notes are delivered to the local coordinator for archiving (we collect this free form feedback anonymously).

Closing the Issue.
